matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/internal/crypto/CryptoStoreTest.kt

```diff
@@ -37,6 +39,8 @@ private const val DUMMY_DEVICE_KEY = "DeviceKey"
 @RunWith(AndroidJUnit4::class)
 class CryptoStoreTest : InstrumentedTest {
 
+    @get:Rule var rule = RetryTestRule(3)
```

Comment:
This could be `val` instead of `val` (= immutable). Applicable to all the occurrences on this PR

Reply:
:+1: Have updated these.